### DISCUSSION TITLE
Is there any way of individually fixing column width?

### DISCUSSION THREAD

**Discussion by nirmalviper** [Power-Components/livewire-powergrid#967] [Q&A]
# ⚡ PowerGrid ⚡ Feature Request

## Summary

Brief description of what is at stake.

### Why is this needed?

A clear and concise description of the problem.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/request

**Comment by luanfreitasdev** [comment]
Have you already taken a look here?

https://v4.livewire-powergrid.com/table/include-columns.html#headerattribute

**Comment by ashek1412** [comment]
hi, I tried header headerAttribute and bodyAttribute, but nothing happned. is there way to do that

**Comment by MimisK13** [comment]
Was a solution finally found?

**Comment by lozynskiadam** [comment]
This solution works for me:

```
Column::make('Id', 'id')
    ->headerAttribute(styleAttr: 'width: 150px;')
    ->sortable(),
```
